src/languages/autoit.js

```diff
@@ -1,22 +1,40 @@
 /*
 Language: AutoIt
 Author: Manh Tuan <junookyo@gmail.com>
+Modified: Michał Lipok
```

Comment:
```suggestion
```

We don't generally track this per-file like this other than main author and HUGE rewrites.